src/common/Tile/styled.js

```diff
@@ -198,6 +198,7 @@ export const MovieDescription = styled.p`
   font-size: 20px;
   line-height: 1.6;
   text-align: justify;
+  margin-left: -20px;
```

Comment:
tak patrzę i nie widzę różnicy i potrzeby zastosowania tego? Co to zmienia?

Reply:
ten margin? nie wiem, to nie moja zmiana tylko merge z mainem, więc nie ruszam. Może ktoś miał w tym zamysł.

Reply:
Dokładnie tutaj to ja dodawałem jest to potrzebne przy floacie ;)